backend/routers/plugins.py

```
@@ -109,3 +119,104 @@ def get_plugin_money_made(plugin_id: str):
+# @router.get('/v1/migrate-plugins', tags=['v1'])
+# def migrate_plugins():
+#     response = requests.get('https://raw.githubusercontent.com/BasedHardware/Omi/main/community-plugins.json')
+#     if response.status_code != 200:
+#         return []
+#     data = response.json()
+#     for plugin in data:
+#         add_plugin_script(plugin)
+
+
+@router.post('/v3/plugins', tags=['v1'])
+def add_plugin(plugin_data: str = Form(...), file: UploadFile = File(...), uid=Depends(auth.get_current_user_uid)):
```

Ruff

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)